src/cpp/include/openvino/genai/whisper_pipeline.hpp

```diff
@@ -34,6 +34,22 @@ class OPENVINO_GENAI_EXPORTS ChunkStreamerBase : public StreamerBase {
 using ChunkStreamerVariant =
     std::variant<std::function<bool(std::string)>, std::shared_ptr<ChunkStreamerBase>, std::monostate>;
 
+struct OPENVINO_GENAI_EXPORTS WhisperRawPerfMetrics {
+    /** @brief Duration for each features extraction call */
+    std::vector<MicroSeconds> features_extraction_durations;
+};
```

Comment:
I followed parent's `RawPerfMetrics` pattern here. It might be easier to read if derived class implemented in the same manner. I don't mind to get rid of `WhisperRawPerfMetrics`.